tuist/tuist
# Fix project header attributes

### Short description 📝

- Headers with a "Project" visibility were getting a "PROJECT" attribute in their build file in the generated project
- Project headers in Xcode however have no attributes set when set manually in the UI
- As it happens, setting any arbitrary value defaults to "Project" visibility in Xcode hence why this worked thus far

### Solution 📦

To make generated project more inline with Xcode, we can omitt setting attributes for project headers.

### Implementation 👩‍💻👨‍💻

- [x] Update `BuildPhaseGenerator`

### Test Plan 🛠

- Run `tuist generate` within `fixtures/ios_app_with_frameworks/Framework2`
- Verify the generated project has `MyProjectClass.h` set to "Project" visbility in the generated project
